store/store.go

```diff
@@ -50,6 +51,12 @@ const (
 	ModeForceSync
 )
 
+var (
+	RewardToMiner = common.StringToAddress("MINER REWARD")
+	RewardToUncle = common.StringToAddress("UNCLE REWARD")
```

Comment:
Please refer https://github.com/getamis/eth-indexer/blob/develop/model/types.go#L29

Reply:
i think we should add these to types.go

Reply:
move to `model/types.go`